src/transaction/utils.ts

```diff
@@ -412,7 +417,136 @@ function convertMemo(memo: Memo): { Memo: ApiMemo } {
   };
 }
 
+/**
+ * Autofills fields in a transaction.
+ *
+ * @param {Transaction} tx A transaction to autofill fields.
+ * @param {Client} client A client.
+ * @returns {Transaction} An autofilled transaction.
+ */
+async function autofillTransaction(
+  tx: Transaction,
+  client: Client
+): Promise<Transaction> {
+  validateAccountAddress(tx, 'Account', 'SourceTag');
+  if (tx['Destination']) {
```

Comment:
```suggestion
  if (tx['Destination'] == null) {
```

Reply:
This should be `!= null` instead since it's checking if Destination is defined.